src/groups/bmq/bmqp/bmqp_protocol.h

```diff
@@ -3515,7 +3515,7 @@ inline bool MessagePropertiesInfo::isExtended() const
 inline MessagePropertiesInfo::SchemaIdType
 MessagePropertiesInfo::schemaId() const
 {
-    return d_schemaWireId >> 1;
+    return static_cast<SchemaIdType>(d_schemaWireId >> 1);
```

Comment:
Is casting to 'SchemaIdType' better practice than casting to 'unsigned short' in this case?